spec/jekyll_seo_tag_spec.rb

```diff
@@ -196,6 +196,60 @@
       end
     end
 
+    context "with image.path, image.height, and image.width" do
```

Comment:
Alright. Should we save the testing refactor for a separate PR? It seems like a lot of the tests need to be refactored. It appears fall outside of the scope that I was hoping to accomplish with this PR. I'm not against it, but it's going to get harder to refactor with other PRs getting accepted before this one with all of the changes being made.